deps/reif/reif.pl

```diff
@@ -0,0 +1,281 @@
+:- module(reif,
```

Comment:
This was downloaded from 
http://www.complang.tuwien.ac.at/ulrich/Prolog-inedit/swi/reif.pl

Was thinking it'd be really cool to use chop to manage the dependencies of itself 🤓. At least once we have the basics bootstrapped.